src/patterns/ec2-app.test.ts

```diff
@@ -349,15 +349,19 @@ describe("the GuEC2App pattern", function () {
     const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
 
     expect(json.Mappings).toEqual({
-      stagemapping: {
-        CODE: expect.objectContaining({
+      testguec2app: {
```

Comment:
The `app` in this test is `"test-gu-ec2-app"` (defined just above). CDK removes `-`.